packages/time-picker/test/dom/time-picker.test.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync } from '@vaadin/testing-helpers';
+import '../../src/vaadin-time-picker.js';
+
+describe('vaadin-time-picker', () => {
+  let timePicker;
+
+  // Ignore generated attributes to prevent failures
+  // when running snapshot tests in a different order
+  const SNAPSHOT_CONFIG = {
+    ignoreAttributes: ['id', 'aria-describedby', 'aria-labelledby', 'for']
+  };
+
+  beforeEach(() => {
+    timePicker = fixtureSync('<vaadin-time-picker></vaadin-time-picker>');
+  });
+
+  describe('shadow', () => {
+    it('default', async () => {
+      await expect(timePicker).shadowDom.to.equalSnapshot();
+    });
+
+    it('disabled', async () => {
+      timePicker.disabled = true;
+      await expect(timePicker).shadowDom.to.equalSnapshot();
+    });
+
+    it('readonly', async () => {
+      timePicker.readonly = true;
+      await expect(timePicker).shadowDom.to.equalSnapshot();
+    });
+
+    it('invalid', async () => {
+      timePicker.invalid = true;
+      await expect(timePicker).shadowDom.to.equalSnapshot();
+    });
+
+    it('theme', async () => {
+      timePicker.setAttribute('theme', 'align-right');
+      await expect(timePicker).shadowDom.to.equalSnapshot();
+    });
+  });
+
+  describe('slots', () => {
+    it('default', async () => {
+      await expect(timePicker).lightDom.to.equalSnapshot(SNAPSHOT_CONFIG);
+    });
+
+    it('helper', async () => {
+      timePicker.helperText = 'Helper';
+      await expect(timePicker).lightDom.to.equalSnapshot(SNAPSHOT_CONFIG);
+    });
```

Comment:
Would it make sense to add a test for `errorMessage` as well?

Reply:
Done.